File: 1512-number-of-good-pairs/1512-number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int goodPair =  0;
        int len = nums.length;
        for (int i = 0; i < len; i++) {
            int temp = nums[i];
            for (int j = i + 1; j < len; j++) {
                if (temp == nums[j]) {
                    goodPair++;
                }
            }
        }
        return goodPair;
    }
}

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        count=Counter(nums)
        c=0
        for i in count:
            if count[i]<2:
                c+=0
            if count[i]==2:
                c+=1
            if count[i]>2:
                c+=sum(range(0,count[i]))
        return c